vindi-woocommerce-subscriptions.php

```diff
@@ -173,25 +181,49 @@ public function sync_vindi_user_information($user_id, $address_type)
         }
 
         /**
-		 * Show pricing fields at admin's product page.
-		 */
-        public function subscription_custom_fields()
+         * Show pricing fields at admin's product page.
+         */
+        public function simple_subscription_custom_fields()
+        {
+            global $post;
+
+            $plans          = $this->settings->api->get_plans();
+            $selected_plan  = get_post_meta($post->ID, 'vindi_subscription_plan', true);
+            $wc_product     = wc_get_product($post->ID);
+            $product_type   = $wc_product->get_type();
+            $plans['names'] = array(__('-- Selecione --', VINDI_IDENTIFIER)) + $plans['names'];
+
+            $this->settings->get_template(
+                'admin-simple-product-subscription-fields.html.php',
+                compact(
+                    'plans',
+                    'selected_plan',
+                    'product_type'
+                )
+            );
+        }
+
+        /**
+         * Show pricing fields at admin's variable product page.
+         */
+        public function variable_subscription_custom_fields($loop, $variation_data, $variation)
         {
             global $post;
 
             $plans         = $this->settings->api->get_plans();
-    		$selected_plan = get_post_meta($post->ID, 'vindi_subscription_plan', true);
+            $selected_plan = get_post_meta($variation->ID, 'vindi_variable_subscription_plan', true);
 
-            $plans['names'] = array(__('-- Selecione --', VINDI_IDENTIFIER)) + $plans['names'];
+            $plans['names'] = array(__('-- Plano da Vindi Padrão  --', VINDI_IDENTIFIER)) + $plans['names'];
```

Comment:
Que tal plano padrão da Vindi?

```suggestion
            $plans['names'] = array(__('-- Plano padrão da Vindi --', VINDI_IDENTIFIER)) + $plans['names'];
```

Reply:
Perfeito.